src/gui/mzroll/isotopeplot.cpp

```diff
@@ -1,27 +1,24 @@
 #include "isotopeplot.h"
 
-// import most common Eigen types
-//USING_PART_OF_NAMESPACE_EIGEN
 using namespace Eigen;
 
+IsotopePlot::IsotopePlot(QCustomPlot* customPlot, float width, float height, float stackingZValue, 
+                vector<mzSample*> samples, float abundanceThresold, PeakGroup::QType qtype){
```

Comment:
@IpankajI Don't you think every argument should not be part of the constructor?

Reply:
Right but here it's all parameter have to be  passed, as these need to be accessed at many places.  We can pass parameters as method arguments if these are only used by that method only otherwise pass these at time of creating instance of that class.